templates/default/sql/dml/0.10.0.sql.erb

```diff
@@ -0,0 +1 @@
+REPLACE INTO variables VALUES ("spark_avro_package_version", "<%= @spark_avro_package_version %>");
```

Comment:
Please don't do this. In another PR I already fix the `featurestore_default_storage_format` and `featurestore_default_storage_format`.
Replace this line with 
` REPLACE INTO variables VALUES ("spark_avro_package_version", "<%= node['hadoop_spark']['spark_avro_package_version'] %>"); ` and it will work just fine.
Also, remove the variable in the default recipe.

Reply:
Ok